ts/commands/PackageCommands.ts

```diff
@@ -42,14 +42,21 @@ class PackageCommands {
       });
 
     (caporal as any)
-      .command('package install', 'Install a package from a Zip URL')
+      .command('package install', 'Install a package from a Zip URL or File')
       .option('--username', 'Metricly Username')
       .option('--password', 'Metricly Password')
       .option('--profile', 'Metricly profile', /.*/, 'default')
-      .argument('<url>', 'Package Download URL')
+      .argument('[url]', 'Package Download URL')
```

Comment:
How do you access the second argument without specifying an initial URL argument?

Reply:
right, that's not going to work

Reply:
You could take one arg and use https://www.npmjs.com/package/is-url to tell if it's URL or not.

Reply:
I like the idea of letting it detect whether it's a url and fallback to file.  Last night I changed the args to options.  LMK, I think I could be convinced to let it detect vs needing the options.